cloud/azure/sfx/integrations-azure.tf

```diff
@@ -39,6 +39,7 @@ resource "signalfx_azure_integration" "azure_integration" {
 
 # "Named token to use for ingest on the SignalFx Azure integration"
```

Comment:
```suggestion
# "Named token to use for ingest on the SignalFx Azure integration"
# We need a specific token to avoid using the Default organization one.
```